server/jetstream_cluster_3_test.go

```diff
@@ -6433,3 +6433,98 @@ Consume3:
 		t.Errorf("Consumers to same stream are at different sequences: %d vs %d", a, b)
 	}
 }
+
+func TestJetStreamAccountFileStoreLimits(t *testing.T) {
```

Comment:
Naming convention here may mean that Travis will not run this test. We need to check that.